<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="tests.property.IntegerTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2023-11-30T18:26:06" hostname="i57pclanzinger" time="1.09">
  <properties/>
  <testcase name="run[/home/florian/git/property-checker-public/tests/property/IntegerTest]" classname="tests.property.IntegerTest" time="1.09">
    <failure message="java.lang.AssertionError: 12 out of 12 expected diagnostics were found.&#10;9 unexpected diagnostics were found:&#10;  IntTest.java:42: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:54: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:55: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:56: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:60: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:67: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:70: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:73: error: (type.invalid.annotations.on.location)&#10;  IntTest.java:75: error: (type.invalid.annotations.on.location)&#10;While type-checking /home/florian/git/property-checker-public/tests/property/IntegerTest/IntTest.java" type="java.lang.AssertionError">java.lang.AssertionError: 12 out of 12 expected diagnostics were found.
9 unexpected diagnostics were found:
  IntTest.java:42: error: (type.invalid.annotations.on.location)
  IntTest.java:54: error: (type.invalid.annotations.on.location)
  IntTest.java:55: error: (type.invalid.annotations.on.location)
  IntTest.java:56: error: (type.invalid.annotations.on.location)
  IntTest.java:60: error: (type.invalid.annotations.on.location)
  IntTest.java:67: error: (type.invalid.annotations.on.location)
  IntTest.java:70: error: (type.invalid.annotations.on.location)
  IntTest.java:73: error: (type.invalid.annotations.on.location)
  IntTest.java:75: error: (type.invalid.annotations.on.location)
While type-checking /home/florian/git/property-checker-public/tests/property/IntegerTest/IntTest.java
	at org.junit.Assert.fail(Assert.java:89)
	at org.checkerframework.framework.test.TestUtilities.assertTestDidNotFail(TestUtilities.java:521)
	at org.checkerframework.framework.test.CheckerFrameworkRootedTest.checkResult(CheckerFrameworkRootedTest.java:31)
	at org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest.checkResult(CheckerFrameworkPerDirectoryTest.java:47)
	at org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest.run(CheckerFrameworkPerDirectoryTest.java:164)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:112)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:40)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:60)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:52)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:176)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[Wrote file /IntTest.java with: 
	13 assertions (to be proven in JML)
	8 assumptions (proven by checker)
	1 non-free method preconditions (to be proven in JML)
	3 free method preconditions (proven by checker)
To see the javac command line and output, run with: -Pemit.test.debug
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
